events/events.go

```diff
@@ -81,22 +81,17 @@ func WatchEventStatus(ctx context.Context, client kubernetes.Interface, namespac
 	}
 	kch := w.ResultChan()
 	ch := make(chan *EventStatus, 2)
+	initialTimestamp := metav1.Now()
```

Comment:
should this also filter based on namespace in addition to time? There can multiple topos deployed in the same cluster only differentiated by namespace

Reply:
The namespace filtering is done by line 79

Reply:
aha, that was cutoff in my view. thanks